### ISSUE TITLE
Test (JUnit) - LockAssert.assertLocked

### ISSUE THREAD

**Issue by ChaudhryAtif** [lukas-krecan/ShedLock#221]
I have the following code:

```
@Scheduled(...)
@SchedulerLock(name = "scheduledTaskName")
public void scheduledTask() {
   LockAssert.assertLocked();
   // do something
}
```

When I run my unit test for scheduledTask, I get an error:
`java.lang.IllegalStateException: The task is not locked.`

Is there an example code as to how we can go about mocking and testing other aspects of this library?

**Comment by lukas-krecan** [comment]
Thanks for feedback. This is interesting, I will think about a solution, in the meantime the best thing you can do is to create your own abstraction around `assertLocked()` and switch the assert off in the unit tests.

**Comment by lukas-krecan** [comment]
Would something like this help? https://github.com/lukas-krecan/ShedLock/pull/222

**Comment by lukas-krecan** [comment]
Released as 4.7.0
